src/web3/WalletClient.ts

```diff
@@ -69,7 +80,12 @@ export class WalletClient extends BaseClient implements IWalletClient {
     this.getAccountBalance = this.getAccountBalance.bind(this);
   }
 
-  /** set the default (base) account */
+  /**
+   * Set the default (base) account
+   *
+   * @param baseAccount - The account to set as default
+   * @return a promise that resolves when the base account is set
```

Comment:
```suggestion
   *
   * @return a promise that resolves when the base account is set
```

Reply:
Maybe a @see IAccount ?

Reply:
Well maybe not on this particular method but at least on the class description, because it manipulates this type and we might not know anything about it the first time we use it that's the point of links to related informations/classes I think.